labs/triggering-deployments.md

```diff
@@ -0,0 +1,56 @@
+# Triggering Deployments
+Deploying container images can happen, manually or automatically.  In this lab, you'll do both: a manual deployment through the Spinnaker UI and an automatic deployment using the Container Builder Build Trigger we set up previously.
+
+
+## Running your pipeline manually
+1. Return to the Pipelines page by clicking on the “Pipelines” button
+1. Click on the “Start Manual Execution” button
+    
+    ![](../docs/img/image7.png)
+
+1. Select the v1.0.0 tag from the dropdown, then click Run
+  ![](../docs/img/image17.png)
+1. Once the pipeline starts, click the “Details” button to see more information about the build’s progress. 
+    > **Note:** This section shows the status of the deployment pipeline and its steps. Steps in blue are currently running, green ones have completed successfully and red ones have failed. By clicking on one of the stages, you can see more details about its progress.
+  
+    ![](../docs/img/image3.png)
+  
+1. After about 3-5 minutes the integration test phase will be complete and the pipeline will require a manual approval to continue the deployment. Hover over the Yellow person icon and select “Continue”.
+  ![](../docs/img/image8.png)
+1. Your roll out will continue to the production frontend and backend deployments and will complete after a few minutes. 
+1. To view the app, first click on “Load Balancers” in the top right of the Spinnaker UI
+  ![](../docs/img/image16.png)
+1. Scroll down the list of Load Balancers and click on Default, under “sample-frontend-prod”
+  ![](../docs/img/image6.png)
+1. Scroll down the details pane on the right and click the clipboard button ![](../docs/img/image9.png) on the Ingress to copy your apps IP address.
+  ![](../docs/img/image4.png)
+1. Paste the address into your browser to view the app:
+  ![](../docs/img/image13.png)
+
+
+## Trigger your pipeline from code changes
+Now that your pipeline is running as expected when started manually, you can try triggering it automatically when code changes are pushed. Rather than building after every new commit, you will run it only when new tags are pushed to your Git repository. You have set up your Container Builder trigger to kick off an image build and push it any time a Git tag prefixed with “v” is pushed to your repo. Once the image is successfully pushed, your Spinnaker pipeline will detect that a new image exists in your Container Registry with a “v” prefix and run the pipeline with that image. The same build artifact, your Docker image, will be used across all phases of deployment from unit tests, to canary deployment and finally to your production services.
```

Comment:
We can also just take out this text, point to the solution once its published for more detail. WDYT?